### ISSUE TITLE
Update Clause 12 on media types

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/ogc-i3s-community-standard#5]
Clause 12 appears to only have placeholder text inside it.

It should be updated with a statement that specifies the associated media types, updated with text that says that “This version of the I3S Community Standard does not specify any media types for data encodings”.

**Comment by cnreediii** [comment]
Agreed -  Double checked that there are no media type references in content. Added sentence as suggested.

**Comment by ghobona** [comment]
This Issue is related to https://github.com/opengeospatial/ogcapi-3d-geovolumes/issues/13

@cnreediii It appears that there is a media type that is mentioned in one of the profile documents.

https://github.com/opengeospatial/ogc-i3s-community-standard/blob/d56247271d314077a34ede6173c62e90cddcc367/docs/store.cmn.adoc?plain=1#L113

On a separate note, I would recommend consistent use of `I3S` vs `i3s` in the segment of the media type.

https://github.com/opengeospatial/ogc-i3s-community-standard/blob/d56247271d314077a34ede6173c62e90cddcc367/docs/store.cmn.adoc?plain=1#L55

**Comment by cnreediii** [comment]
@ghobona Thanks Gobe. There was a "niggling" in my brain that there was a media type expressed but I did not check the profiles yesterday. Will check and fix.

**Comment by cnreediii** [comment]
@ghobona Actually a bit more :-) In store.psl, the vnd is actually referenced to the "ogc" and not "esri". Also, I missed changing version numbers from 1.2 to 1.3. Made all those changes.

**Comment by cnreediii** [comment]
Done!
